src/one_dragon/utils/i18_utils.py

```
@@ -6,16 +6,20 @@
-            return 'zh'
-        else:
-            return 'en'
+        system_locale = locale.getlocale()[0]
+        if system_locale:
+            system_language = system_locale.lower()
+            if system_language.startswith('zh'):
+                return 'zh'
+            if system_language.startswith('ja'):
+                return 'ja'
+        return 'en'
     except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)